contracts/interchain-token-service/src/state.rs

```diff
@@ -144,6 +144,7 @@ pub fn load_chain_config(
 pub fn save_chain_config(
     storage: &mut dyn Storage,
     chain: &ChainNameRaw,
+    its_contract: Address,
     max_uint: nonempty::Uint256,
     max_target_decimals: u8,
```

Comment:
Consider 
```suggestion
    config: impl Into<ChainConfig>
```
and implement `From<msg::ChainConfig> for ChainConfig`.